cmd/manager/main.go

```diff
@@ -29,8 +31,9 @@ import (
 
 // Change below variables to serve metrics on different host or port.
 var (
-	metricsHost       = "0.0.0.0"
-	metricsPort int32 = 8383
+	metricsPath               = "/metrics"
+	metricsPort               = "8080"
+	secretWatcherScanInterval = time.Duration(10) * time.Minute
```

Comment:
This can be removed (I think this got copied over from aws-account-operator on accident)